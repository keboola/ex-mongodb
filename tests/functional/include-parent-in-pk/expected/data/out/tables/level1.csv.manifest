{"incremental":false,"columns":["a","b","level2-a","level2-b","root_pk"],"column_metadata":{"a":[{"key":"KBC.datatype.nullable","value":true}],"b":[{"key":"KBC.datatype.nullable","value":true}],"level2-a":[{"key":"KBC.datatype.nullable","value":true}],"level2-b":[{"key":"KBC.datatype.nullable","value":true}],"root_pk":[{"key":"KBC.datatype.nullable","value":true}]}}