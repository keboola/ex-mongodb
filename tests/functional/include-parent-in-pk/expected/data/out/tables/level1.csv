"123","123","ad1c2165423aa6c30ae7b2515625a1ab","ad1c2165423aa6c30ae7b2515625a1ab","abd3a376c3391349acdd8314c86d52c2"
"123","123","cf90539a89e1696793d8f8c29b89ca61","cf90539a89e1696793d8f8c29b89ca61","242e5b0bd72bbe997aa19b77707bae5a"
"456","123","77d54caf02384eea2f166dff9cdb70c1","77d54caf02384eea2f166dff9cdb70c1","da9abe6c21ccf7b9eb54d279b0b70326"
"456","123","854dab3d9cfa9d5994da2ddeec1e9446","854dab3d9cfa9d5994da2ddeec1e9446","17622ddee10ab75bd3de2eb4d96532c1"
